libmcount/internal.h

```diff
@@ -415,6 +415,8 @@ struct mcount_disasm_engine {
 #define INSTRUMENT_FAILED                       -1
 #define INSTRUMENT_SKIPPED                      -2
 
+#define MAX_COND_BRANCH 3
```

Comment:
Please add a comment why it cannot be more than 3.

Reply:
Since the size of the smallest jcc is 2 bytes, at most we can replace 3 of them by the call in the trampoline. This may be true for x86, maybe not for other archs. Knowing that this file is also arch-agnostic, what is the recommended value to use ?

Reply:
I think 3 is ok.